SU-SWS/stanford_jumpstart_academic
# HSDO-1110: added rules and a table to email and record user changes

# READY FOR REVIEW

# Summary
- Added rules to email alex and dustin in the event of a site owner user changes
- Added a database table to log all user update activity.

# Needed By (Date)
- Turkey Day

# Urgency
- Mild

# Steps to Test

1. Checkout this branch
2. enable and configure [reroute_email](https://www.drupal.org/project/reroute_email) to prevent unwanted emails to dustin and alex
3. enable stanford_jsa_user_rules module
4. edit a user and add/remove the site owner role.
5. ensure your rerouted email works.
6. View the table in the database `stanford_jsa_user_changes`. No UI is added for this. Its entirely for manual administrative inspection.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
